en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,185 @@
+
+- Updated occurrence: result indicator period actual @value attribute (`#358 <https://github.com/IATI/IATI-Schemas/issues/358>`__) (`discussion link <https://discuss.iatistandard.org/t/results-represent-more-than-quantitative-data-included-2-03/872>`__)
+
+- Added new document-link element (as a child of various result elements)(`#359 <https://github.com/IATI/IATI-Schemas/issues/359>`__) (`discussion link <https://discuss.iatistandard.org/t/add-document-link-to-results-indicator-included-2-03/895>`__)
+ 
+- Added new reference element (as a child of the result element) (`#362 <https://github.com/IATI/IATI-Schemas/issues/362>`__) (`discussion link <https://discuss.iatistandard.org/t/results-vocabulary-attribute-option-included-2-03/879>`__)
+ 
+- Added new @aggregation-status attribute for the indicator element (`#363 <https://github.com/IATI/IATI-Schemas/issues/363>`__) (`discussion link <https://discuss.iatistandard.org/t/results-improve-consistency-of-results-standard-included-2-03/874>`__)
+ 
+- Added new location element (as a child of the baseline element) (`#365 <https://github.com/IATI/IATI-Schemas/issues/365>`__) (`discussion link <https://discuss.iatistandard.org/t/results-improve-consistency-of-results-standard-included-2-03/874>`__)
+ 
+- Added new dimension element (as a child of the baseline element) (`#366 <https://github.com/IATI/IATI-Schemas/issues/366>`__) (`discussion link <https://discuss.iatistandard.org/t/results-allow-disaggregations-of-results-data-included-2-03/871>`__)
+  
+Organisations Schema
+-------------------
+
+- Updated definition: organisation-identifier element (`#361 <https://github.com/IATI/IATI-Schemas/issues/361>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+
+Activities and Organisations Schemas
+-------------------
+  
+- Updated definition: all @xml:lang attributes (`#336 <https://github.com/IATI/IATI-Schemas/issues/336>`__) (`discussion link <https://discuss.iatistandard.org/t/language-recommend-use-of-iso-639-1-included-2-03/842>`__)
+
+- Updated definition: [reporting/participating/receiver/provider]-org @ref attribute (`#339 <https://github.com/IATI/IATI-Schemas/issues/339>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+
+- Updated definition: reporting-org @secondary-reporter attribute (`#340 <https://github.com/IATI/IATI-Schemas/issues/340>`__) (`discussion link <https://discuss.iatistandard.org/t/modify-definition-of-secondary-publisher-included-2-03/846>`__)
+ 
+- Updated definition: [recipient-region/recipient-region-budget/sector] @vocabulary-uri attribute (`#341 <https://github.com/IATI/IATI-Schemas/issues/341>`__) (`discussion link <https://discuss.iatistandard.org/t/guidance-on-u-r-i-usage-for-publisher-s-own-vocabularies-included-2-03/850>`__)
```

Comment:
The issue title and contents will need to be amended to also include the definition change for `iati-activity/transaction/recipient-region/@vocabulary-uri`, which appeared to have been missed - see https://github.com/IATI/IATI-Schemas/pull/391#issuecomment-365568200

Reply:
Is this a second attribute that will have the definition updated in the same way, but via a second PR? If so, this should probably be a second line in the changelog to reference the second PR number.

Reply:
The attribute was missed from the original list in #341 - the list has now been updated.

The changelog in this PR (and in extra documentation changes) links to issue numbers (not PRs) so there is no need to link to the second PR since it itself is referenced in the log within #341.

Reply:
Fixed in eb5119d.